wagtail/admin/templates/wagtailadmin/shared/editing_sessions/module.html

```diff
@@ -0,0 +1,46 @@
+{% load i18n wagtailadmin_tags %}
+
+{% dialog id="w-unsaved-changes-dialog" icon_name="warning" icon_classname="w-text-text-error" title=_("Refreshing the page means you will lose any unsaved changes") %}
+    <div class="w-editing-sessions-dialog-buttons">
+        <button class="button" type="button" data-controller="w-action" data-action="w-action#forceReload">{% trans "Refresh page" %}</button>
+        <button class="button button-secondary" type="button" data-action="w-dialog#hide">{% trans "Cancel" %}</button>
+    </div>
+{% enddialog %}
+
+{% blocktranslate trimmed with user_name=_("System") model_name=content_type.name asvar someone_has_saved_message %}
+    {{ user_name }} has saved a newer version of this {{ model_name }}
+{% endblocktranslate %}
+
+{% blocktranslate trimmed with user_name=_("System") asvar overwrite_message %}
+    Proceeding will overwrite the changes made by {{ user_name }}. Refreshing the page will lose any of your unsaved changes.
+{% endblocktranslate %}
```

Comment:
I don’t understand in what scenario those messages would display. Are there any scenarios where they’re _not_ overwritten by the ones in `EditingSessionsList`?

Reply:
No, there aren't any intended scenarios (unless the dialog is triggered by custom code). I need to pass non-empty strings to the `{% dialog %}` tag so it would render the appropriate elements. I could change this to something like "Placeholder" if that works better?